modules/apps/jaggery/emm/src/emm/modules/backend-service-invoker.js

```diff
@@ -166,13 +166,13 @@ var backendServiceInvoker = function () {
      * @param errorCallback a function to be called if en error is reserved.
      * @param soapVersion soapVersion which need to used.
      */
-    function initiateWSRequest(action, endpoint, payload, successCallback, errorCallback, soapVersion) {
+    privateMethods.initiateWSRequest = function (action, endpoint, payload, successCallback, errorCallback, soapVersion) {
         var ws = require('ws');
         var wsRequest = new ws.WSRequest();
         var options = new Array();
         if (IS_OAUTH_ENABLED) {
-            var accessToken = getAccessToken();
-            if (!(!accessToken)) {
+            var accessToken = privateMethods.getAccessToken();
```

Comment:
accessToken is used after the validation.so no need to call a methoad for a second time.